In [1]:
import pandas as pd

df =  pd.read_csv("GreenSpaces.csv")
df

_id Responsibility_Parks or Operations                        Facility  \
0     1                         Operations               Bates St Triangle   
1     2                         Operations            Baum Negley Triangle   
2     3                         Operations                   Bedford Strip   
3     4                         Operations              Beechview Monument   
4     5                         Operations              Brookline Monument   
5     6                         Operations                  Craft Triangle   
6     7                         Operations                       Crosstown   
7     8                         Operations              East Liberty Strip   
8     9                         Operations                 Edgerton Circle   
9    10                         Operations        Fort Duquesne Blvd Strip   
10   11                         Operations         Frankstown Triangle 1&2   
11   12                         Operations    Glen Arden Circle & Triangle   
12   13                         Operations            Grant Street Islands   
13   14                         Operations                 Harris Triangle   
14   15                         Operations        Highland Ave Plant Strip   
15   16                         Operations                   Jail Triangle   
16   17                         Operations                Le Roi Rd Island   
17   18                         Operations           Liberty Avenue Island   
18   19                         Operations            McPherson Blvd Strip   
19   20                         Operations              Mellon Park Island   
20   21                         Operations  Monitor St and St Phil Islands   
21   22                         Operations                Negley Run Strip   
22   23                         Operations         Negley-Collins Triangle   
23   24                         Operations                  Oakland Square   
24   25                         Operations                Parkview Islands   
25   26                         Operations            Polish Hill Monument   
26   27                         Operations      Regent Square War Memorial   
27   28                         Operations     Reynolds St Planting Circle   
28   29                         Operations               Sheraden Monument   
29   30                         Operations          Swisshelm War Memorial   
30   31                         Operations      Thomas Blvd Planting Strip   
31   32                         Operations                  Vietnam Island   
32   33                         Operations         Windermere Circle 1 & 2   
33   34                         Operations             Zulema Passive Area   
34   35                                NaN                             NaN   

    Division  Total Acreage  Maintainable Acreage  Shelter  Rec Center  \
0        3.0           0.20                   NaN      NaN         NaN   
1        2.0           0.20                   NaN      NaN         0.0   
2        3.0           0.30                   NaN      NaN         0.0   
3        5.0           0.10                   NaN      NaN         NaN   
4        4.0           0.10                   NaN      NaN         0.0   
5        3.0           0.10                   NaN      NaN         NaN   
6        6.0           1.20                   NaN      NaN         NaN   
7        2.0           2.00                   NaN      NaN         NaN   
8        3.0           0.01                   NaN      NaN         0.0   
9        6.0            NaN                   NaN      NaN         0.0   
10       2.0           1.80                   NaN      NaN         0.0   
11       3.0           0.01                   NaN      NaN         0.0   
12       6.0            NaN                   NaN      NaN         0.0   
13       3.0           0.10                   NaN      NaN         0.0   
14       2.0           0.05                   NaN      NaN         NaN   
15       6.

In [2]:
dictf =  pd.read_csv("dictionary.csv")
dictf

_id                           Column Name  \
0     1    Responsibility_Parks or Operations   
1     2                              Facility   
2     3                              Division   
3     4                         Total Acreage   
4     5                  Maintainable Acreage   
5     6                               Shelter   
6     7                            Rec Center   
7     8                         Senior Center   
8     9                                  Pool   
9    10                            Spray Pool   
10   11                             Play Area   
11   12              Tennis Court with Lights   
12   13        Tennis Court without NO Lights   
13   14     Basketball Full Court with Lights   
14   15  Basketball Full Court without Lights   
15   16     Basketball half Court with Lights   
16   17  Basketball Half Court without Lights   
17   18                  Basketball Standards   
18   19                   Multi-Purpose Court   
19   20                           Funnel Ball   
20   21            Baseball Field with Lights   
21   22         Baseball Field without lights   
22   23                       Volleball Court   
23   24                          Boccie Court   
24   25                          Hockey Court   
25   26                             Horseshoe   
26   27                        Drinking Fount   
27   28                          Neighborhood   
28   29                          Census Tract   
29   30                             Bleachers   
30   31                          Council Dist   
31   32                               CD Area   
32   33                                 Grill   
33   34                    Decorative Foutain   
34   35                             Flag Pole   
35   36                            Grandstand   
36   37    Control Link for Basketball Courts   
37   38         Control Link for Tennis Court   
38   39        Control Linkl for Hockey Court   
39   40                      Dek Hockey Court   
40   41                              Dog Park   
41   42                            Spray Park   
42   43                 Ownership of Facility   
43   44                               Benches   
44   45                                Tables   

                                            Definiton  
0                                                 NaN  
1                                           Park name  
2   Division within the Department of Public Works...  
3                    Total area of the park, in acres  
4          Area of the park that requires maintenance  
5                                  Number of shelters  
6                        Number of recreation centers  
7                            Number of senior centers  
8                            Number of swimming pools  
9                               Number of spray pools  
10                 Number of play areas with children  
11       Number of tennis courts equipped with lights  
12             Number of tennis courts without lights  
13  Number of full basketball courts equipped with...  
14    Number of full basketball courts without lights  
15  Number of half-sized basketball courts equippe...  
16  Number of half-sized basketball courts without...  
17           Number of basketball hoops on the courts  
18                     Number of multi-purpose courts  
19                        Number of funnel ball games  
20     Number of baseball fields equipped with lights  
21           Number of baseball fields without lights  
22                        Number of volleyball courts  
23                       Number of boccie ball courts  
24                            Number of hockey courts  
25                            Number of horshoe games  
26                       Number of drinking fountains  
27                   Neighborhood that the park is in  
28                               Federal Census Tract  
29                      Number of bleachers installed  
30      Commu

Neighborhoods of Parks
1. Central Oakland
2. Friendship
3. Bedford Dwelling
4. Beechview
5. Brookline
6. West Oakland
7. Central Business District
8. East Liberty 
9. Point Breeze
10. Central Business District
11. Homewood South
12. Point Breeze
13. Central Business District
14. Crafton Heights
15. Highland Park
16. South Oakland
17. Point Breeze North
18. Central Business District
19. Point Breeze North
20. Shadyside
21. Squirrel Hill North
22. East Liberty
23. East Liberty
24. Central Oakland
25. South Oakland
26. Polish Hill
27. Regent Square
28. Point Breeze
29. Sheraden
30. Swisshelm Park
31. Homewood West
32. Shadyside
33. Swisshelm Park
34. Central Oakland

# Total Acres:
Central Oakland - 1.3
Friendship - 0.2
Bedford Dwelling - 0.3
Beechview - 0.1
Brookline - 0.1
West Oakland - 0.1 
Central Business District - 1.2 
East Liberty - 2.1
Point Breeze - 0.31
Homewood South - 1.8
Crafton Heights - 0.1
Highland Park - 0.05
South Oakland - 0.14
Point Breeze North - 0.8 
Shadyside - 0.8
Squirrel Hill North - 0.3
South Oakland - 0.1 
Polish Hill - 0.01
Sheraden - 0.1
Swisshelm Park - 0.05
Homewood West - 0.7

From this submetric the best green neighborhood would be East Liberty. Unfortunately this data does not include all of the large parks such as Schenly and Highland, and many other small green spaces throughout the area of Pittsburgh. Meaning that even though the data says East Liberty is most green neighborhood and many neighborhoods do not have any green at all is technically not true. These parks are just the ones collected by the Operations Division of the Department of Public Works.